aten/src/ATen/native/sparse/SoftMax.cpp

```diff
@@ -291,10 +292,10 @@ void cpu_sparse_coo_softmax(Tensor output, const Tensor& input, const int64_t di
   if (dim >= sparse_dim) {
     if (LogSoftMax) {
       auto new_values = log_softmax_cpu(values, dim - sparse_dim + 1, false);
-      out_values.copy_(new_values);
+      out_values.set_(new_values);
```

Comment:
NOTE: this is only OK if there's no user-facing API where the user can provide an out tensor. In those cases we want to preserve the storage.